### ISSUE TITLE
最新版本使用docker安装脚本部署后无法启动

### ISSUE THREAD

**Issue by sh6x-228** [blinko-space/blinko#280]
在debian系统下，使用安装脚本

`curl -s https://raw.githubusercontent.com/blinko-space/blinko/main/install.sh | bash`

安装一切正常，然后blinko-website容器无法启动，docker logs查看日志显示：

```Error: Could not parse schema engine response: SyntaxError: Unexpected token E in JSON at position 0
prisma:warn Prisma failed to detect the libssl/openssl version to use, and may not work as expected. Defaulting to "openssl-1.1.x".
Please manually install OpenSSL and try installing Prisma again.
prisma:warn Prisma failed to detect the libssl/openssl version to use, and may not work as expected. Defaulting to "openssl-1.1.x".
Please manually install OpenSSL and try installing Prisma again.
Prisma schema loaded from schema.prisma
Datasource "db": PostgreSQL database "postgres" at "blinko-postgres:5432"

Error: Could not parse schema engine response: SyntaxError: Unexpected token E in JSON at position 0
prisma:warn Prisma failed to detect the libssl/openssl version to use, and may not work as expected. Defaulting to "openssl-1.1.x".
Please manually install OpenSSL and try installing Prisma again.
prisma:warn Prisma failed to detect the libssl/openssl version to use, and may not work as expected. Defaulting to "openssl-1.1.x".
Please manually install OpenSSL and try installing Prisma again.
Prisma schema loaded from schema.prisma
Datasource "db": PostgreSQL database "postgres" at "blinko-postgres:5432"

Error: Could not parse schema engine response: SyntaxError: Unexpected token E in JSON at position 0
```

install脚本内容如下

`cat install.sh`

```#!/bin/bash

# Colors for better visibility
RED='\033[0;31m'
GREEN='\033[0;32m'
YELLOW='\033[1;33m'
NC='\033[0m' # No Color

# Check if the network 'blinko-network' already exists
if [ ! "$(docker network ls -q -f name=blinko-network)" ]; then
    echo -e "${YELLOW}Network 'blinko-network' does not exist. Creating network...${NC}"
    docker network create blinko-network

    if [ $? -ne 0 ]; then
        echo -e "${RED}Failed to create Docker network. Please check your Docker setup.${NC}"
        exit 1
    fi
    echo -e "${GREEN}Successfully created Docker network: blinko-network${NC}"
else
    echo -e "${YELLOW}Network 'blinko-network' already exists. Skipping network creation.${NC}"
fi

# Step 2: Check if the PostgreSQL container 'blinko-postgres' already exists
if [ "$(docker ps -aq -f name=blinko-postgres)" ]; then
    echo -e "${YELLOW}Container 'blinko-postgres' already exists. Skipping container creation.${NC}"
else
    echo -e "${YELLOW}2.   Starting PostgreSQL container...${NC}"
    docker run -d \
      --name blinko-postgres \
      --network blinko-network \
      -p 5435:5432 \
      -e POSTGRES_DB=postgres \
      -e POSTGRES_USER=postgres \
      -e POSTGRES_PASSWORD=mysecretpassword \
      -e TZ=Asia/Shanghai \
      --restart always \
      postgres:14

    if [ $? -ne 0 ]; then
      echo -e "${RED}Failed to start PostgreSQL container.${NC}"
      exit 1
    fi
    echo -e "${GREEN}✅ PostgreSQL container is running.${NC}"
fi

# Step 3: Prompt user to optionally mount the .blinko directory
echo -e "${YELLOW}Do you want to mount a local '.blinko' directory to '/app/.blinko' in the container? (y/n)${NC}"
read -p "Enter your choice: " mount_choice

if [[ "$mount_choice" == "y" || "$mount_choice" == "Y" ]]; then
    read -p "Please provide the path to your '.blinko' folder: " blnko_folder

    # Check if the directory exists; if not, create it
    if [ ! -d "$blnko_folder" ]; then
        echo -e "${YELLOW}Directory does not exist. Creating directory...${NC}"
        mkdir -p "$blnko_folder"

        if [ $? -ne 0 ]; then
            echo -e "${RED}Failed to create the directory. Please check permissions.${NC}"
            exit 1
        fi
    fi

    # Check if the directory has write permissions
    if [ ! -w "$blnko_folder" ]; then
        echo -e "${RED}The directory '$blnko_folder' does not have write permissions.${NC}"
        exit 1
    fi

    echo -e "${GREEN}Directory is ready for mounting: $blnko_folder${NC}"
    volume_mount="-v $blnko_folder:/app/.blinko"
else
    volume_mount=""
    echo -e "${YELLOW}Skipping mounting of .blinko directory.${NC}"
fi

# Step 4: Run BlinkOS container with or without volume path
echo -e "${YELLOW}3. ️️ Starting BlinkOS container...${NC}"
docker run -d \
  --name blinko-website \
  --network blinko-network \
  -p 1111:1111 \
  -e NODE_ENV=production \
  -e NEXTAUTH_SECRET=my_ultra_secure_nextauth_secret \
  -e DATABASE_URL=postgresql://postgres:mysecretpassword@blinko-postgres:5432/postgres \
  $volume_mount \
  --restart always \
  blinkospace/blinko:latest

if [ $? -ne 0 ]; then
  echo -e "${RED}Failed to start BlinkOS container.${NC}"
  exit 1
fi

echo -e "${GREEN}✅ All containers are up and running.${NC}"
```

**Comment by youran** [comment]
me too, it suddenly stop working today.
I didn't change the docker-compose file.

**Comment by blinko-space** [comment]
yes, The nodejs docker could not find the openssl dependency due to alpine 3.21 moving some libraries. This build issue has been fixed on v0.21.12 

https://github.com/nodejs/docker-node/issues/2175
